server/config/config.go

```diff
@@ -409,6 +412,27 @@ func (c *Config) Parse(arguments []string) error {
 				c.Log.Level = c.LogLevelDeprecated
 			}
 		}
+		if c.EnableLocalTSO {
```

Comment:
why do we still keep EnableLocalTSO, TSOSaveInterval, and TSOUpdatePhysicalInterval in the server config after we added TSOConfig tso.Config?

Reply:
OK, I know the reason. It's to make toml.Decode(Config) happy.